lib/Util/TextFormat.php

```diff
@@ -18,6 +18,10 @@ public function indent(string $string, int $level = 0)
     {
         $lines = explode(PHP_EOL, $string);
         $lines = array_map(function ($line) use ($level) {
```

Comment:
test is missing for this class, but it might be good to add

Reply:
Done